### PR TITLE
Hotfix/fix kids club checkin family finder issues

### PR DESCRIPTION
**Pull request by dtkocher**
Fix for KC Sort Order and Guest Flag.

**Build:** http://mp-ci.centralus.cloudapp.azure.com/viewType.html?buildTypeId=Code_Release_HotFix&branch_Code_Release=hotfix%2FFixKidsClubCheckinFamilyFinderIssues&tab=buildTypeStatusDiv

### CODE REVIEW

**Comment by jpc11623** [comment]
Reviewed, looks like what we need.

File: CI/SQL/20170419_061700_US7920-add-guest-flag-column.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+-- ===============================================================
+
+USE [MinistryPlatform]
+GO
+
+IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE Name = N'Guest_Sign_In' AND Object_ID = Object_ID(N'dbo.Event_Participants'))
+BEGIN
+	ALTER TABLE dbo.Event_Participants ADD
+		Guest_Sign_In BIT NULL
+END
```
Comment (k8tied1) [inline CI/SQL/20170419_061700_US7920-add-guest-flag-column.sql]: Is there going to be another PR to set/use this column? I checked in crds-signin-checkin and there isn't a PR for release.
